src/python/zquantum/core/cost_function.py

```diff
@@ -89,15 +91,15 @@ class EvaluateOperatorCostFunction(CostFunction):
 
     Args:
         target_operator (openfermion.QubitOperator): operator to be evaluated
-        ansatz (dict): dictionary representing the ansatz
+        ansatz (zquantum.core.interfaces.ansatz.Ansatz): ansatz usef to evaluate cost function
```

Comment:
Spelling mistake ;)

Reply:
Addressed in [73b8143](https://github.com/zapatacomputing/z-quantum-core/pull/42/commits/73b8143e1c5870115ce7f99c9d94978711159e66)